src/DynamoCoreWpf/Views/Gallery/GalleryView.xaml

```diff
@@ -0,0 +1,116 @@
+﻿<Window x:Class="Dynamo.Wpf.Views.Gallery.GalleryView"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+        xmlns:ui="clr-namespace:Dynamo.UI"
+        ShowInTaskbar="False"
+        ResizeMode="NoResize"
+        WindowStartupLocation="CenterOwner"
+        WindowStyle="SingleBorderWindow"
+        Height="600" Width="800">
+    
+    <Window.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Window.Resources>
+    
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="100"></ColumnDefinition>
+            <ColumnDefinition Width="600"></ColumnDefinition>
+            <ColumnDefinition Width="100"></ColumnDefinition>
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="50"></RowDefinition>
+            <RowDefinition Height="300"></RowDefinition>
+            <RowDefinition Height="50"></RowDefinition>
+            <RowDefinition Height="50"></RowDefinition>
+            <RowDefinition Height="*"></RowDefinition>
+        </Grid.RowDefinitions>
+
+        <Button x:Name="CloseButton" 
+                HorizontalAlignment="Right" 
+                BorderThickness="0,0,0,0" 
+                Grid.Column="1"
+                Background="Black"
+                Grid.Row="0"
+                Width="32" 
+                Height="32" 
+                VerticalAlignment="Bottom" 
+                Click="CloseGallery">
+            <Button.Content>
+                <Image Source="CloseIcon.png" 
+                       VerticalAlignment="Center" 
+                       HorizontalAlignment="Center" />
+            </Button.Content>
+        </Button>
+
+        <Button x:Name="LeftButton" 
+                HorizontalAlignment="Center" 
+                BorderThickness="0,0,0,0" 
+                Grid.Column="0" 
+                Grid.Row="1"
+                Background="Black" 
+                VerticalAlignment="Center" 
+                Width="32" 
+                Height="32" 
+                Command="{Binding Path=MovePrevCommand}" >
+            <Image Source="LeftIcon.png" 
+                   VerticalAlignment="Center" 
+                   HorizontalAlignment="Center" />
+        </Button>
+
+        <Image x:Name="currentImage" 
+               Source="{Binding Path=CurrentImage, UpdateSourceTrigger=PropertyChanged}" 
```

Comment:
Can you test without `UpdateSourceTrigger=PropertyChanged` for all bindings in this file?